CounterpointGenerator/CounterpointGenerator/Rules/IRules.cs

```diff
@@ -0,0 +1,10 @@
+namespace CounterpointGenerator {
+
+    /**
+     * Base interface for rules objects
+     */
+    public interface IRules {
+        public List<int> apply(List<int> possibilities);
```

Comment:
Fix imports to add List

Reply:
(This applies to all the files, need to add the right references so that the code will compile)

Reply:
Resolved in latest push